src/SprykerEco/Client/Payone/PayoneClientInterface.php

```diff
@@ -89,7 +89,7 @@ public function cancelRedirect(PayoneCancelRedirectTransfer $cancelRedirectTrans
      *
      * @api
      *
-     * @param \Generated\Shared\Transfer\PayonePaymentDirectDebitTransfer $bankAccountCheckTransfer
+     * @param \Generated\Shared\Transfer\PayoneBankAccountCheckTransfer $bankAccountCheckTransfer
```

Comment:
- Please add the `- Specification` mark to the method document block.
- Check other methods.

Reply:
Fixed